ABloom/ABloom/Presentation/CoreViews/QnA/View/QuestionMainView.swift

```diff
@@ -56,15 +29,50 @@ struct QuestionMainView: View {
             AnswerCheckView(num: content)
           }
         })
-        
-        .frame(maxWidth: .infinity, maxHeight: .infinity)
         .background(backWall())
       }
       .background(backgroundDefault())
     }
   }
 }
 
+extension QuestionMainView {
+  private var headerView: some View {
+    // 헤더
+    HStack {
+      Text("우리의 문답")
+        .fontWithTracking(.title3Bold)
+        .padding([.leading], 20)
+      Spacer()
+      // FIXME: 다른 뷰로 전환 시에는 다른 방식으로 처리해야함
+      NavigationLink(value: 0) {
+        Image("pencil_write_fill")
+          .padding([.trailing], 17)
+      }
```

Comment:
```suggestion
      NavigationLink(value: 0) {
        Image("pencil_write_fill")
      }
      .padding([.trailing], 17)
```
이렇게 수정하면 패딩으로된 여백을 클릭했을 때 화면이동되는 것을 방지할 수 있을 것 같아요!